boot/seal.go

```diff
@@ -47,6 +47,27 @@ var (
 	seedReadSystemEssential = seed.ReadSystemEssential
 )
 
+// Hook function setup by devicestate to support device-specific full
+// disk encryption implementations.
+var HasFDESetupHook = func(*BootableSet) bool {
```

Comment:
I forgot to ask, why does this one get the bootable set?

Reply:
It really just needs the kernel info from that set, so I could change the signature.

Reply:
I'm still not sure how that will work but probably easier to tweak if needed when we get further than to discuss/change now.